.gitignore

```diff
@@ -16,6 +16,7 @@ pom.xml
 .nrepl-history
 .rebel_readline_history
 .idea
+*.iml
```

Comment:
what's this about?

Reply:
@NoahTheDuke This was part of the development wiki to gitignore IDE build files

Reply:
Those are for intellij metadata? Can't hurt to add it I think.

Reply:
per your comment on another PR I have added it to exclude and removed it from gitignore